test/feature-benchmark/scenarios.py

```diff
@@ -1482,3 +1482,43 @@ def benchmark(self) -> BenchmarkingSequence:
 """
             ),
         ]
+
+
+class HydrateIndex(ScenarioBig):
+    """Measure the time it takes for an index to hydrate when a cluster comes online."""
+
+    def init(self) -> list[Action]:
+        return [
+            self.table_ten(),
+            TdAction(
+                """
+> CREATE CLUSTER idx_cluster SIZE '16', REPLICATION FACTOR 1
+> CREATE TABLE t1 (f1 INTEGER, f2 INTEGER)
+"""
+            ),
+        ]
+
+    def benchmark(self) -> MeasurementSource:
+        return Td(
+            f"""
+> SET CLUSTER = default
+> DROP INDEX IF EXISTS i1
+> DELETE FROM t1
+> ALTER CLUSTER idx_cluster SET (REPLICATION FACTOR 0)
+> CREATE INDEX i1 IN CLUSTER idx_cluster ON t1(f1)
+> INSERT INTO t1 (f1) SELECT {self.unique_values()} FROM {self.join()}
```

Comment:
Can you please run the feature-benchmark from the Release Qualification  pipeline? It runs all the scenarios with `--scale=+1` . Changing the INSERTS above from 1 per timestamp to 10 in 1 timestamp may cause the `self.join()` trick to take up too much memory.

The other option is to use `generate_series` and see if this works out better

Reply:
Done, will wait for result: https://buildkite.com/materialize/release-qualification/builds/363 Locally it's fine with ~8 GB memory peak, so should be fine.